share/disambiguation.py

```diff
@@ -127,22 +120,19 @@ def _instance_for_node(self, node):
 
         for q in constrain:
```

Comment:
I'm not sure... if there's a person with the same name as an institution, the loose constraints will match both, I think? And we don't want to merge them together. There's likely a simpler way to do this, but theres [a ticket](https://openscience.atlassian.net/browse/SHARE-549) for that.